kivy/core/camera/camera_avfoundation.pyx

```diff
@@ -58,14 +59,25 @@ class CameraAVFoundation(CameraBase):
         cdef char *data
         cdef char *metadata_type
         cdef char *metadata_data
+        cdef cyview.array cyarr
 
         if self.stopped:
             return
 
+        
```

Comment:
was this added empty line intentional? it doesn't seem pep8 compliante

Reply:
Absolutely not, I was doing some debug in that area, and I missed it during cleanup. Removed.